### ISSUE TITLE
Long group names are shown in next line in html_repr

### ISSUE THREAD

**Issue by percygautam** [arviz-devs/arviz#1248]
The `html_repr` for `InferenceData` needs following modifications:

- Long group names like `predictions_constant_data` and warmup group names are shown in line to the marker
- If the `InferenceData` object has only one group, show the group in expanded form.

Nothing urgent. Just to remind myself later.

**Comment by OriolAbril** [comment]
I think we can use two classes to get everything to work, currently group names are class `xr-section-summary`, we can add an `arviz-group` class to override the `grid-template-columns` and define the group names as 

```
<label class="xr-section-summary arviz-group" ...>...</label>
```

**Comment by OriolAbril** [comment]
I kind of remember @sethaxen fixing this but would like confirmation before closing.

**Comment by sethaxen** [comment]
I believe this was fixed in #1407.
